.github/workflows/run_one_off.yml

```diff
@@ -0,0 +1,141 @@
+name: run_one_off
```

Comment:
I recommend a name that reflects the contents or frequency of the test suite (if we expect the contents to change often), e.g. `run_examples`,  `run_post-merge_tests` (follows `run_tests` naming, or `post-merge_tests`

Reply:
renamed to run_post_merge_tests.yml